config/initializers/content_security_policy.rb

```diff
@@ -0,0 +1,25 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I'll file a ticket here to see if we can strip `secure_headers`. This is a great enhancement by team rails.